src/main/java/Task/Event.java

```diff
@@ -0,0 +1,68 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import java.util.Locale;
+
+
+/**
+ * Represents an event task in the task manager application.
+ * An event task has a description, start time, and end time.
+ */
+public class Event extends Task{
+    private LocalDateTime startTime;
+    private LocalDateTime endTime;
+    private static final String LINE = "\t______________________________________________________";
+    private DateTimeFormatter inputFormat = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+    private DateTimeFormatter outputFormat = DateTimeFormatter.ofPattern("MMM dd yyyy HHmm", Locale.ENGLISH);
+
+    /**
+     * Constructs an Event task with the specified description, start time, and end time.
+     *
+     * @param description The description of the event task.
+     * @param startTime The start time of the event in the format "dd/MM/yyyy HHmm".
+     * @param endTime The end time of the event in the format "dd/MM/yyyy HHmm".
+     */
+    public Event(String description, String startTime, String endTime) {
+        super(description);
+        this.startTime = LocalDateTime.parse(startTime, inputFormat);
+        this.endTime = LocalDateTime.parse(endTime, inputFormat);
+    }
+
+    @Override
+    public String print() {
+        String str = "";
+        try {
+            String start = startTime.format(outputFormat);
+            String end = endTime.format(outputFormat);
+            str = "[E]" + super.print() + "(from: " +
+                    start + ") (to: " + end + ")";
+        } catch (DateTimeParseException e) {
+            System.out.println(LINE);
+            System.out.println("\t I think you haven't had enough vitamin C."
+                    + "\n\t Your time format should be :"
+                    + "\n\t\t { dd/MM/yyyy HHmm }"
+                    + "\n\t I suggest you take some LEMONA.");
+            System.out.println(LINE);
+        }
+        return str;
+    }
+
+    @Override
+    public String getDescription() {
+        String start = startTime.format(outputFormat);
+        String end = endTime.format(outputFormat);
+        String str = "[E] " + super.getDescription() + " " + start + " " + end;
+        return str;
+    }
+
+    @Override
+    public String getTaskInfo() {
+        String start = startTime.format(outputFormat);
+        String end = endTime.format(outputFormat);
```

Comment:
For more descriptive variable name, you may rename `start` and `end` into `formattedStartTime` and `formattedEndTime`.